docs/how-to/configure-tls-encryption.md

```diff
@@ -0,0 +1,165 @@
+# TLS encryption in COS
+
+For both COS and COS Lite, the following statements detail the default TLS implementation:
+
+- COS charms generate CSRs with the K8s FQDN as the SAN DNS and the internal CA signs.
+- All COS charms trust the internal CA by installing the CA certificate in the charm and workload containers, using the `update-ca-certificates` tool.
+- The external CA provides a certificate for Traefik's external URL.
+- Within the COS model, workloads communicate via K8s FQDN URLs.
```

Comment:
Not really true: for example, grafana datasources are integrated via ingress; grafana-agent writes to Mimir/Loki/Tempo via ingress.

The internal communication of the HA solutions instead goes over FQDN.